pynucastro/templates/amrexastro-cxx-microphysics/tfactors.H.template

```diff
@@ -0,0 +1,34 @@
+#ifndef TFACTORS_H
+#define TFACTORS_H
+
+struct tf_t {
```

Comment:
we create `tfactors.H.template` to contain the `tf_t` struct. (I wonder why we consider this a .template file)

Reply:
the way the C++ output works is that it searches for all the files that end in `.template` and it copies them into the output directory, filling in any information that is needed.